src/core/unit.c

```diff
@@ -2740,6 +2740,8 @@ int unit_serialize(Unit *u, FILE *f, FDSet *fds, bool serialize_jobs) {
         if (u->cgroup_path)
                 unit_serialize_item(u, f, "cgroup", u->cgroup_path);
         unit_serialize_item(u, f, "cgroup-realized", yes_no(u->cgroup_realized));
+        fprintf(f, "cgroup-realized-mask=%i\n", u->cgroup_realized_mask);
+        fprintf(f, "cgroup-enabled-mask=%i\n", u->cgroup_enabled_mask);
```

Comment:
hmm, i am note sure i like this bit: it means the precise bit positions become API, and I am not sure we want that, given that kernel controllers are added (and even removed) all the time. I'd prefer if we could instead store this via the controller names, using cgroup_controller_to_string() and cgroup_controller_from_string().

I figure this means we should have unit_serialize_cgroup_mask() or so, which generates the right output.